pkg/framework/machinesets.go

```diff
@@ -109,6 +120,75 @@ func CreateMachineSet(c client.Client, params MachineSetParams) (*machinev1.Mach
 	return ms, nil
 }
 
+// BuildMachineSetParamsList creates a list of MachineSetParams based on the given machineSetParams with modified instance type.
+func BuildAlternativeMachineSetParams(machineSetParams MachineSetParams, platform configv1.PlatformType) []MachineSetParams {
+	baseMachineSetParams := machineSetParams
+	baseProviderSpec := baseMachineSetParams.ProviderSpec.DeepCopy()
+
+	output := []MachineSetParams{}
+	switch platform {
+	case configv1.AWSPlatformType:
+		// Using cheapest compute optimized instances that meet openshift minimum requirements (4 vCPU, 8GiB RAM)
+		alternativeInstanceTypes := []string{"c5.xlarge", "c5a.xlarge", "m5.xlarge"}
```

Comment:
Just thought: m6i.large also might be added here, I think.

Reply:
I think you meant the current default m6i.xlarge as m6i.large only has two cores. I think using different instance types for the spot test will give lower interruption rate.